### PR TITLE
inspect and list command updated

### PR DESCRIPTION
**Pull request by davidsanfal**
* conan inspect info added
* conan list ... --format=html info added

### CODE REVIEW

File: reference/commands/inspect.rst [hunk 1]
```diff
@@ -23,3 +23,125 @@ conan inspect
                             -vvv or -vtrace
       --logger              Show the output with log format, with time, type and
                             message.
+
+Inspect default and json output
+-------------------------------
```
Comment (franramirez688) [inline reference/commands/inspect.rst:28]: I think the title is not necessary here.

File: reference/commands/inspect.rst [hunk 2]
```diff
@@ -23,3 +23,127 @@ conan inspect
                             -vvv or -vtrace
       --logger              Show the output with log format, with time, type and
                             message.
+
+Inspect default and json output
```
Comment (memsharded) [inline reference/commands/inspect.rst:27]: Please tag ``conan inspect`` as preview, it was always a bit problematic command

File: reference/commands/inspect.rst [hunk 3]
```diff
@@ -23,3 +23,127 @@ conan inspect
...
+-------------------------------
+
+This ``conan inspect`` default output shows the following information format.
```
Comment (memsharded) [inline reference/commands/inspect.rst:30]: It is worth adding an "important" note that says that ``conan inspect`` doesn't really evaluate any methods or apply any conditional logic. It list class attributes only.

File: reference/commands/list.rst [hunk 4]
```diff
@@ -346,3 +346,38 @@ with the following structure:
...
+
+.. code-block:: text
+
+  $ conan list "zlib/1.2.12#*:*" --format=html
+  <!DOCTYPE html>
+  <html lang="en">
+    <head>
+      <title>conan list results</title>
+      <script>
+        var list_results = {"Local Cache": {"zlib/1.2.12": {"revisions": {"c2cd5ec484e443e4f01b51ff5370fd11": {"timestamp": 1665747741.12, "packages": {"6fe7fa69f760aee504e0be85c12b2327c716f9e7": {"info": {"settings": {"arch": "x86_64", "build_type": "Release", "compiler": "apple-clang", "compiler.version": "14", "os": "Macos"}, "options": {"fPIC": "True", "shared": "False"}}}}}, "0de8ff7f99079cd07341311c9ead89a2": {"timestamp": 1676913747.767964, "packages": {"6fe7fa69f760aee504e0be85c12b2327c716f9e7": {"info": {"settings": {"arch": "x86_64", "build_type": "Release", "compiler": "apple-clang", "compiler.version": "14", "os": "Macos"}, "options": {"fPIC": "True", "shared": "False"}}}}}}}}};
```
Comment (memsharded) [inline reference/commands/list.rst:364]: I think the specifics of the output file are irrelevant. I'd document that you can open it, or maybe do a screen capture (or embed the output html) to display the actual result

File: reference/commands/inspect.rst [hunk 5]
```diff
@@ -23,3 +23,127 @@ conan inspect
...
+        msa: [True, False]
+        sse: [True, False]
+        vsx: [True, False]
+        api_prefix: ['ANY']
+    revision_mode: hash
+    settings: ('os', 'arch', 'compiler', 'build_type')
+    topics: ('png', 'graphics', 'image')
+    url: https://github.com/conan-io/conan-center-index
+
+The ``conan inspect ... --format=json`` will return a json output in ``stdout`` with the same information.
```
Comment (franramirez688) [inline reference/commands/inspect.rst:62]:
```suggestion
The :command:`conan inspect ... --format=json` returns a JSON output format in ``stdout`` (can be redirected to file) with the following structure::
```

File: reference/commands/inspect.rst [hunk 6]
```diff
@@ -23,3 +25,128 @@ conan inspect
                             -vvv or -vtrace
       --logger              Show the output with log format, with time, type and
                             message.
+
+.. note::
+
+    ``conan inspect`` doesn't really evaluate any methods or apply any conditional logic. It list class attributes only.
```
Comment (RubenRBS) [inline reference/commands/inspect.rst:31]:
```suggestion
    ``conan inspect`` doesn't really evaluate any methods or apply any conditional logic. It lists class attributes only.
```

File: reference/commands/inspect.rst [hunk 7]
```diff
@@ -23,3 +25,128 @@ conan inspect
...
+        msa: [True, False]
+        sse: [True, False]
+        vsx: [True, False]
+        api_prefix: ['ANY']
+    revision_mode: hash
+    settings: ('os', 'arch', 'compiler', 'build_type')
+    topics: ('png', 'graphics', 'image')
+    url: https://github.com/conan-io/conan-center-index
+
+The :command:`conan inspect ... --format=json` returns a JSON output format in ``stdout`` (can be redirected to file) with the following structure::
```
Comment (RubenRBS) [inline reference/commands/inspect.rst:65]:
```suggestion
The :command:`conan inspect ... --format=json` returns a JSON output format in ``stdout`` (which can be redirected to a file) with the following structure::
```

File: reference/commands/list.rst [hunk 8]
```diff
@@ -345,3 +345,43 @@ with the following structure:
       }
     }
   }
+
+List html output
+----------------
+
+The ``conan list ... --format=html`` will return a html output in ``stdout`` (can be redirected to file)
```
Comment (RubenRBS) [inline reference/commands/list.rst:352]:
```suggestion
The ``conan list ... --format=html`` will return a html output in ``stdout`` (which can be redirected to a file)
```

**Review by RubenRBS — APPROVED** [review]
:)

**Pull request merged**